index.js

```diff
@@ -1,3 +1,3 @@
 const batteryBatches = [4, 5, 3, 4, 4, 6, 5];
 
-// Code your solution here
+const totalBatteries = batteryBatches.reduce((battery, currentTotal) => battery + currentTotal, 0);
```

Comment:
This works! But to be more accurate, you would flip the argument names. 
```suggestion
const totalBatteries = batteryBatches.reduce((currentTotal, battery) => battery + currentTotal, 0);
```